src/adapter/breakpoints.ts

```diff
@@ -178,7 +179,7 @@ export class Breakpoint {
     this._setUrlLocations.add(urlLocation);
 
     const location = {
-      url,
+      url: fixDriveLetter(url, false),
```

Comment:
CDP seems to (at least on my machine...) always want lowercased drive letters. @EricCornelson I'm curious if you ran into this before/whether it's a platform specific thing.

Reply:
I hadn't noticed that as an issue with CDP, but I have noticed vscode is case-sensitive (even on windows) when matching paths, e.g. when matching a loaded script to a file in the workspace, (some example issues: https://github.com/microsoft/vscode/issues/44074 https://github.com/microsoft/vscode/issues/43959 https://github.com/microsoft/vscode/issues/12448)

At one point I think Rob proposed using https://github.com/microsoft/vscode-uri to help keep things more standardized, but that lib also has its inconsistencies (but at least they would be consistent with vscode 😄 )

Reply:
Diego suggested the `MapUsingProjection` from v2. I'll grab that here.

Reply:
v1 sets the driveletter it in a case-insensitive way: `Debugger.setBreakpointByUrl\",\"params\":{\"urlRegex\":\"file:\\\\/\\\\/\\\\/[Cc]:\\\\/Users\\\\/roblou\\\\/code\\\\...`

Reply:
I think we need that, because the drive letter case in node depends on the windows environment. https://github.com/microsoft/vscode-node-debug2/issues/37

Reply:
Ah, good thought. I'll adjust that, that makes me much more comfortable :)

Reply:
Added in the latest commit